jbpm-services/jbpm-services-api/src/build/revapi-config.json

```diff
@@ -42,6 +53,18 @@
         "old": "field org.jbpm.services.api.query.QueryResultMapper<T>.COLUMN_ORGANIZATIONAL_ENTITY",
         "new": "field org.jbpm.services.api.query.QueryResultMapper<T>.COLUMN_ORGANIZATIONAL_ENTITY",
         "justification": "Fix for some of dbs with query aliases"
+      },
+      {
+        "code": "java.field.enumConstantOrderChanged",
+        "old": "field org.jbpm.services.api.query.model.QueryDefinition.Target.CUSTOM",
+        "new": "field org.jbpm.services.api.query.model.QueryDefinition.Target.CUSTOM",
+        "justification": "Included more out of the box targets for query definition that are more specific than custom and thus placed higher in the enum"
+      },
+      {
+        "code": "java.field.serialVersionUIDUnchanged",
+        "old": "field org.jbpm.services.api.query.model.QueryParam.serialVersionUID",
+        "new": "field org.jbpm.services.api.query.model.QueryParam.serialVersionUID",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
```

Comment:
Do we still wan to wait for this explanation? @MarianMacik I remember we talked about this, but we never came to a conclusion as I am not sure what to put here. But I guess anything is better than "ADD YOUR...". Worst case we could add at least something like "We don't care about that" or "We don't really know why is this being reported" :)